docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md

```diff
@@ -98,14 +98,14 @@ For example, the CHW value at index (c,h,w) is physically located at index (c\*H
 Supported Model Formats
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-==================  ===========================  ============================  ==========================
- Plugin              FP32                         FP16                          I8                              
-==================  ===========================  ============================  ========================== 
- CPU plugin          Supported and preferred      Supported                     Supported                                                        
- GPU plugin          Supported                    Supported and preferred       Supported                                                        
- GNA plugin          Supported                    Supported                     Not supported                                                    
- Arm® CPU plugin     Supported and preferred      Supported                     Supported (partially)                                            
-==================  ===========================  ============================  ==========================
++------------------+--------------------------+--------------------------+------------------------+
+| Plugin           | FP32                     | FP16                     | I8                     |
++==================+==========================+===================================================+ 
+| CPU plugin       | Supported and preferred  | Supported                | Supported              |
+| GPU plugin       | Supported                | Supported and preferred  | Supported              |
+| GNA plugin       | Supported                | Supported                | Not supported          |
+| Arm® CPU plugin  | Supported and preferred  | Supported                | Supported (partially)  |
++------------------+--------------------------+--------------------------+------------------------+
 
 
 For :doc:`Multi-Device <openvino_docs_OV_UG_Running_on_multiple_devices>` and
```

Comment:
Not changed in this PR, but can we delete "*Generally, FP16 is preferable as it is most ubiquitous and performant*.". It is not true. It was never "most ubiquitous" but it is also no longer most performant. Model format is now independent of inference precision; on GPU (where FP16 is most performant) inference will be in FP16 by default regardless of model format. FP16 for now has larger memory requirements than FP32. So the only benefit of FP16 is smaller file size, which at the moment comes at a larger memory cost. IMO It's better to not have a blanket recommendation for now. Memory downside of FP16 will be resolved in the next release.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16913/commits/77a9364a34d8b658d98fab05ab314a2c54c12c54